cdk/lib/dotcom-components.ts

```diff
@@ -92,67 +106,78 @@ chown -R dotcom-components:support /var/log/dotcom-components
-                tableName: `support-admin-console-channel-tests-${this.stage}`,
-            }),
-        ];
-
-        const ec2App = new GuEc2App(this, {
-            applicationPort: 3030,
-            app: appName,
-            access: { scope: AccessScope.PUBLIC },
-            certificateProps: {
-                [Stage.CODE]: {
-                    // TODO: should be dotcom-components.support.code.dev-guardianapis.com
-                    domainName: 'dotcom-components-code.support.guardianapis.com',
-                },
-                [Stage.PROD]: {
-                    domainName: 'dotcom-components.support.guardianapis.com',
-                },
-            },
-            monitoringConfiguration: {
-                http5xxAlarm: {
-                    tolerated5xxPercentage: 0.5,
-                    numberOfMinutesAboveThresholdBeforeAlarm: 1,
-                    alarmName: `URGENT 9-5 - high 5XX error rate on ${this.stage} support-dotcom-components`,
-                },
-                unhealthyInstancesAlarm: true,
-                snsTopicName,
-            },
-            userData,
-            roleConfiguration: {
-                additionalPolicies: policies,
-            },
-            scaling: { PROD: { minimumInstances: 3, maximumInstances: 18 } },
-        });
-
-        ec2App.autoScalingGroup.scaleOnCpuUtilization('CpuScalingPolicy', {
-            targetUtilizationPercent: 40,
-        });
-    }
+			elkStream.valueAsString
+		} /var/log/dotcom-components/dotcom-components.log`;
+
+		const policies: Policy[] = [
+			new GuGetS3ObjectsPolicy(this, 'S3ReadPolicySupportAdminConsole', {
+				bucketName: 'support-admin-console',
+				paths: [
+					`${this.stage}/banner-deploy/*`,
+					`${this.stage}/channel-switches.json`,
+					`${this.stage}/configured-amounts.json`,
+					`${this.stage}/guardian-weekly-propensity-test/*`,
+				],
+			}),
+			new GuGetS3ObjectsPolicy(
+				this,
+				'S3ReadPolicyGuContributionsPublic',
+				{
+					bucketName: 'gu-contributions-public',
+					paths: [
+						`epic/${this.stage}/*`,
+						`header/${this.stage}/*`,
+						`banner/${this.stage}/*`,
+					],
+				},
+			),
+			new GuDynamoDBReadPolicy(this, 'DynamoReadPolicy', {
+				tableName: `super-mode-${this.stage}`,
+			}),
+			// TODO: remove when secondary indexes are included in GuDynamoDBRead
+			new GuDynamoDBReadPolicy(this, 'DynamoReadPolicySecondaryIndex', {
+				tableName: `super-mode-${this.stage}/index/*`,
+			}),
+			new GuPutCloudwatchMetricsPolicy(this),
+			new GuDynamoDBReadPolicy(this, 'DynamoTestsReadPolicy', {
+				tableName: `support-admin-console-channel-tests-${this.stage}`,
+			}),
+		];
+
+		const scaling: GuAsgCapacity = {
+			minimumInstances: this.stage === 'CODE' ? 1 : 3,
+			maximumInstances: this.stage === 'CODE' ? 2 : 18,
+		};
+
+		const ec2App = new GuEc2App(this, {
+			instanceType: InstanceType.of(
+				InstanceClass.T4G,
+				InstanceSize.SMALL,
+			),
```

Comment:
Out of interest, where did this config live before? I don't see it in the "before" view.

Reply:
this one surprised me - it's passed as a cloudformation parameter, though I'm not sure how that was getting used!